packages/marketplace/docs/exchange/Exchange.md

```diff
@@ -113,23 +113,14 @@ If a collection doesn't support the interface `IRoyaltyUGC`, the secondary marke
 
 ### Royalties
 
-The royalties are the share returning to the creator of the collection or token after a sale.
+The royalties are the share returning to the creator (or owner) of the collection or token after a sale.
```

Comment:
by `owner` what do you mean, how is this set? Want to check my understanding

Reply:
it's very relative to how the contract handle its royalties. But most of the time the guy deploying the contract is the one receiving the royalties thus "owner"